lib/honeycomb/span.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-require "securerandom"
 require "forwardable"
+require "securerandom"
```

Comment:
Is this still needed to use `Random`?

Reply:
doesn't look like it, 💥 it!

Reply:
We can also blast it from `trace.rb` as well

Reply:
removed this